modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -370,129 +361,132 @@ If you want to update several processes, simply run the command with all the pro
 ++++
 
+=== 6.x application resources
 
-== Migrating to Java 11 in Bonita 7.9
+6.x application resources have been removed in Bonita 7.8.0, so if you are migrating a process that leverages this feature, you need to modify it (for example to use process dependencies instead (Configure > Process dependencies in Bonita Studio)).
 
-Bonita 7.9+ supports Java 11.
+=== Updating to Java 11 in Bonita 7.9 or a greater version
+
+Bonita 7.9 and greater versions support Java 11.
 Migrating an existing platform to Java 11 is not an easy, or painless endeavour.
 To migrate a Bonita platform to Java 11, you need to follow the following steps:
 
-* Migrate the platform to Bonita 7.9.0 as usual, and keep running it in Java 8. Verify that everything works as expected.
-* Test the migrated platform in Java 11, on a test environment.
-* Once tested, update what is required on the production server, and switch it to Java 11.
+. Update Bonita Runtime to Bonita 7.9.0 as usual, and keep running it in Java 8
+. Verify that everything works as expecte
+. Test the target Runtime in Java 11, on a test environment
+. Update what is required on the production server
+. Switch it to Java 11
 
-The main parts that require attention and testing are connectors and custom code.
-While the 7.9.0 migration step tries its best to migrate the implementation of connectors that are known not to work in Java 11, namely  xref:https://documentation.bonitasoft.com/bonita/7.9/release-notes#_connectors_dependency_updates[WebService, CMIS, Email and Twitter], custom connectors, groovy scripts, rest api extensions etc. are not migrated and might not work outright in Java 11.
-Aside from just code incompatibility, special attention has to be given to the dependencies of the custom code, as they might either not work in Java 11, work fine but conflict with Bonita own dependencies or the script might use dependencies previously included in Bonita, but no more accessible, or accessible in a different version.
+The main parts that require attention and testing are connectors and custom code. +
+While the 7.9.0 updating step tries its best to migrate the implementation of connectors that are known not to work in Java 11, namely  xref:https://documentation.bonitasoft.com/bonita/7.9/release-notes#_connectors_dependency_updates[WebService, CMIS, Email and Twitter], custom connectors, groovy scripts, REST API extensions etc. are not migrated and might not work outright in Java 11. +
+Aside from just code incompatibility, special attention has to be given to the dependencies of the custom code, as they might either not work in Java 11, work fine but conflict with Bonita own dependencies, or the script might use dependencies previously included in Bonita, but no more accessible, or accessible in a different version.
```

Comment:
but no more accessible => but not accessible anymore

Reply:
fixed